### PR TITLE
Moderate bio or username

### PR DESCRIPTION
**Pull request by davidgljay**
## What does this PR do?

Adds the ability to moderate a bio or a username to the admin panel.

## How do I test this PR?

1. Flag a users's bio in the comment stream.
2. Check the 'pending' moderation queue, you should see the flagged bio in the stream.
3. Click 'X', you should see a modal warning you about suspending the user.
4. Click 'Next', you should be able to edit an e-mail to the user.
5. Click 'Send', the user's status should be set to "suspended" and the user should be sent an e-mail notification.

### CODE REVIEW

File: client/coral-admin/src/actions/auth.js [hunk 1]
```diff
@@ -14,7 +14,10 @@ export const checkLogin = () => dispatch => {
       const isAdmin = !!result.user.roles.filter(i => i === 'admin').length;
       dispatch(checkLoginSuccess(result.user, isAdmin));
     })
-    .catch(error => dispatch(checkLoginFailure(error)));
+    .catch(error => {
+      console.error(error);
```
Comment (okbel) [inline client/coral-admin/src/actions/auth.js:18]: extra console

File: client/coral-admin/src/actions/auth.js [hunk 2]
```diff
@@ -14,7 +14,10 @@ export const checkLogin = () => dispatch => {
       const isAdmin = !!result.user.roles.filter(i => i === 'admin').length;
       dispatch(checkLoginSuccess(result.user, isAdmin));
     })
-    .catch(error => dispatch(checkLoginFailure(error)));
+    .catch(error => {
```
Comment (davidgljay) [inline client/coral-admin/src/actions/auth.js:17]: I was getting a lot of unhelpful errors when rendering the page after login success, this ensures that the text from those errors actually appears and is useful for debugging.

File: tests/e2e/pages/adminPage.js [hunk 3]
```diff
@@ -8,7 +8,7 @@ const embedStreamCommands = {
   },
   approveComment() {
     return this
-      .waitForElementVisible('@commentList')
+      .waitForElementVisible('@moderationList')
```
Comment (okbel) [inline tests/e2e/pages/adminPage.js]: 👍 Thanks for adding e2e for this one!

**Review by davidgljay — COMMENTED** [review]
This is intentional. Here's what happened to me a number of times:

1) There's an error in rendering somewhere.
2) The system hits this error when rendering after checkLoginSuccess.
3) Rather than displaying that error, the system triggers this "catch" call which sends a "login_failure" action with no helpful data.

If there's an error on render it seems useful to send it to the console. If there's a genuine error checking login success (and not a rendering error) then sending it to the console prolly won't hurt things.

**Comment by davidgljay** [comment]
Please hold off on reviewing this for the moment, I've found a strange bug caused by merge with master that I want to sort out...

**Comment by davidgljay** [comment]
Ok, brought up to date after changes in some of the recent PRs. Ready for review.

**Comment by okbel** [comment]
@davidgljay Can you merge this with master? It has a conflict Thanks!

**Review by okbel — APPROVED** [review]

File: client/coral-admin/src/components/Comment.js [hunk 4]
```diff
@@ -30,7 +31,18 @@ export default props => {
           {links ?
           <span className={styles.hasLinks}><Icon name='error_outline'/> Contains Link</span> : null}
           <div className={`actions ${styles.actions}`}>
-            {props.modActions.map((action, i) => getActionButton(action, i, props))}
+            {props.modActions.map(
+              (action, i) =>
+              <ActionButton
```
Comment (okbel) [inline client/coral-admin/src/components/Comment.js:36]: Nice! Getting stronger in each PR 💪

**Comment by davidgljay** [comment]
Updated to reflect graphQL changes, ready for review and merge.

File: client/coral-admin/src/components/Comment.js (hunk 4 above)
Comment (davidgljay) [reply client/coral-admin/src/components/Comment.js:36]: Yeah, it's getting used in two places so it makes sense to make it its own component.

**Pull request merged**